backend/smb/connpool.go

```diff
@@ -118,23 +113,20 @@ func (f *Fs) getSessions() int32 {
 }
 
 // Open a new connection to the SMB server.
+//
+// The context is only used for establishing the connection, not after.
 func (f *Fs) newConnection(ctx context.Context, share string) (c *conn, err error) {
-	// As we are pooling these connections we need to decouple
-	// them from the current context
-	bgCtx := context.Background()
```

Comment:
This was not necessary and prevented the context from being used for connection _establishment_. The `go-smb2` library already uses a background context for all operations after the connection is established, unless `WithContext` is used.